lib/linguist/languages.yml

```
@@ -4673,6 +4673,17 @@ Protocol Buffer:
   codemirror_mode: protobuf
   codemirror_mime_type: text/x-protobuf
   language_id: 297
+Protocol Buffer Text Format:
+  type: data
+  aliases:
+  - text proto
+  - protobuf text format
+  extensions:
+  - ".textproto"
+  - ".pbtxt"
+  tm_scope: none
```

Yikes, I just caught this.

```suggestion
  tm_scope: source.textproto
```

`tm_scope` needs to match the [`scopeName`](https://github.com/thejustinwalsh/textproto-grammar/blob/34f4ec92fac956da753c2818f765cb2575cba8de/syntaxes/textproto.tmLanguage.json#L305) property of the grammar to use for highlighting. It's basically a grammar's unique identifier.